src/module/apps/characterImport/gearImport/AmmoParser.ts

```diff
@@ -28,6 +31,14 @@ export class AmmoParser extends BaseGearParser {
             }
         }
 
+        // Assign import flags
+        parsedGear.system.importFlags = genImportFlags(formatAsSlug(chummerGear.name_english), parserType);
+
+        let subType = formatAsSlug(chummerGear.name_english.split(':')[0]);
+        if (Object.keys(SR5.itemSubTypeIconOverrides[parserType]).includes(subType)) {
+            parsedGear.system.importFlags.subType = formatAsSlug(subType);
+        }
```

Comment:
This seems item agnostic. Do you want to move it to the BaseGearParser or BaseParserFunction and call it?

Reply:
The subtype itself takes different formats depending on the item types, so I can’t use the same code for every item type. Some types also don’t have subtypes (contacts, sins, etc.)

Reply:
I've implemented this in https://github.com/smilligan93/SR5-FoundryVTT/pull/764/commits/031ca8b5c5e4e2ed75ca73e8cab7c62435099ece